examples/sample-kmdf-driver/Cargo.toml

```diff
@@ -31,6 +31,9 @@ wdk-sys = { path = "../../crates/wdk-sys", version = "0.3.0" }
 
 [features]
 default = []
+
+hid = ["wdk-sys/hid"]
```

Comment:
`--all-features` should probably be passed to all the CI jobs for now. Eventually, we should probably use something like `cargo-hack` for more thorough testing